research-software-directory/RSD-as-a-service
# 462 organisation back link

# Enable deep linking organisation pages

Closes #462 

Changes proposed in this pull request:
*  The organisation pages software, projects (and others visible to maintainers) can be directly linked by url (deep linking)
*  It enables user to send a url to others witch points directly to list of software/projects of specific organisation

How to test:
* `docker-compose build frontend` to build new image
* `docker-compose up` to start solution
*  optional run data-generator if you do not have data
*  visit organisations page 
* click on one organisation, by default software page is shown
* click on projects, this will create new url with param page=projects
* click on organisations, via bradcrumbs or menu,
* now! use back button, you should be back at the organisation page, at the project overview (where you were before leaving the organisation)

PR Checklist:
